gnomad_qc/v4/variant_qc/vqsr.py

```diff
@@ -1358,14 +1359,14 @@ def main(args):
     parser.add_argument(
         "--snp-features",
         help="Features to use in the SNP VQSR model.",
-        default=VQSR_FEATURES["snv"],
+        default=VQSR_FEATURES["exomes"]["snv"],
         type=str,
         nargs="+",
     )
     parser.add_argument(
         "--indel-features",
         help="Features to use in the indel VQSR model.",
-        default=VQSR_FEATURES["indel"],
+        default=VQSR_FEATURES["exomes"]["indel"],
```

Comment:
was the exome/genome stratification added after this script was run?

Reply:
yup